pages/contact.js

```diff
@@ -0,0 +1,47 @@
+import Head from 'next/head';
+import Image from 'next/image';
+import styles from '../styles/Contact.module.css';
+
+export default function Contact() {
+  return (
+    <div className={styles.container}>
+      <Head>
+        <title>Duguid Construction</title>
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+      <div id={styles.mainContact}>
+        <div id={styles.contactTitle}>
+            <h1 class="h1">Contact Us</h1>
+        </div>
+        <div id={styles.contactInfo}>
+          <div id={styles.opHours}>
+              <h3>Hours of Operation</h3>
+              <p style={{fontSize: '20px'}}>We are open Monday through Friday</p>
```

Comment:
Could we add some sort of caveat to the hours that notes that customers could still reach out and leave a voicemail, shoot an email, etc.?